presto-main/src/main/resources/webapp/index.html

```diff
@@ -114,7 +114,7 @@
 <script type="text/babel" src="assets/query-list.js"></script>
 
 <!-- Fonts -->
-<link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">
+<link href="vendor/roboto-font/font.css" rel="stylesheet">
```

Comment:
Should the file name indicate how the file was generated (`https://fonts.googleapis.com` and `family=Roboto:400,500,700`)  in case someone wants to update the file in the future?

Reply:
Will do. Just try to make it visible (eg a multi-line comment rather than `// ... `). Also, I guess you needed to made some hand modifications of the file, it would be nice to briefly mention them too.

Reply:
Done